### PR TITLE
JDF-144: Add deltaspike-helloworld-jms example

### PR DESCRIPTION
**Pull request by sgilda**
This is @rbenevides version of the helloworld-jms quickstart that uses DeltaSpike configuration properties.

### CODE REVIEW

**Comment by sgilda** [comment]
@rafabene Can you please check this out and make sure I didn't mess it up! :-)

**Comment by rafabene** [comment]
I checked the deltaspike-helloworld-jms and it's ok. I need to change something but better you merge first

**Comment by sgilda** [comment]
Is it something I can change before it's merged? I'd be glad to fix it.

File: deltaspike-helloworld-jms/pom.xml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+            <version>${version.org.jboss.weld}</version>
+        </dependency>
+
+        <!-- Deltaspike API. we use compile scope as we need its API -->
+        <dependency>
+            <groupId>org.apache.deltaspike.core</groupId>
+            <artifactId>deltaspike-core-api</artifactId>
+        </dependency>
+
+        <!-- Deltaspike Impl. we use runtime scope as we its implementation
```
Comment (rafabene) [inline deltaspike-helloworld-jms/pom.xml]: Change the comment to: Deltaspike Impl. We use compile scope as we need its Impl

**Comment by sgilda** [comment]
@pmuir : would you like to review Rafael's quickstart?

**Comment by pmuir** [comment]
Merged

**Pull request closed without merging**